### PR TITLE
Cloud function to send a "TEKKEN TIME" push notification

### PR DESCRIPTION
**Pull request by udiWertheimer**

### CODE REVIEW

File: cloud/notifications.js [hunk 1]
```diff
@@ -12,5 +14,17 @@ module.exports = {
     }).fail(function(err) {
       console.log("error sending notification: " + err.message);
     });
+  },
+
+  notifyInvitation: function() {
+    var everyoneQuery = new Parse.Query(Parse.Installation);
```
Comment (giladgo) [inline cloud/notifications.js]: not needed

**Comment by giladgo** [comment]
:cactus:

File: cloud/notifications.js [hunk 2]
```diff
@@ -12,5 +14,15 @@ module.exports = {
     }).fail(function(err) {
       console.log("error sending notification: " + err.message);
     });
+  },
+
+  notifyInvitation: function() {
+    return Parse.Push.send({
+      where: new Parse.Query(Parse.Installation),
```
Comment (yonbergman) [inline cloud/notifications.js]: Dry this everyone query

**Comment by yonbergman** [comment]
K

**Comment by yonbergman** [comment]
:fire

**Comment by giladgo** [comment]
:poodle:

**Pull request merged**